apps/smart-contracts/core/test/utils.ts

```diff
@@ -126,3 +128,36 @@ export async function getPermitFromSignature(
     s: signature.s,
   }
 }
+
+export async function revertIfNotRoleHolder(
+  roleGetter: Promise<string>,
+  populatedTransactionPromise: Promise<PopulatedTransaction>
```

Comment:
```suggestion
  rolePromise: Promise<string>,
  populatedTransactionPromise: Promise<PopulatedTransaction>
```
naming consistency, since `const role` should `= await rolePromise`